dev-requirements.txt

```diff
@@ -1,5 +1,6 @@
 alabaster==0.7.9
 coverage==4.3.1
+coverage_pth==0.0.1
```

Comment:
The sole purpose of this is to run `import coverage; coverage.process_startup()` at the right time and place, is that right?